README.md

```diff
@@ -12,7 +12,45 @@ Some metagenome and metatranscripome pairs are true pairs that were extracted fr
 
 ## Getting started with this repository
 
-TBD
+This repository uses snakemake to run the pipeline and conda to manage software environments and installations.
+You can find operating system-specific instructions for installing miniconda [here](https://docs.conda.io/en/latest/miniconda.html).
+After installing conda and [mamba](https://mamba.readthedocs.io/en/latest/), run the following command to create the pipeline run environment.
+
+```
+mamba env create -n mtx_mgx --file environment.yml
+conda activate mtx_mgx
+```
+
+To start the pipeline, run:
+```
+snakemake --use-conda -j 2
+```
+
+### Running this repository on AWS
+
+This repository was executed on an AWS EC2 instance (Ubuntu 22.04 LTS ami-085284d24fe829cd0, t2.large, 100 GiB root storage).
```

Comment:
I would specify that it's EBS storage and the type of EBS storage (most likely `gp2`)

Reply:
```suggestion
This repository was executed on an AWS EC2 instance (Ubuntu 22.04 LTS ami-085284d24fe829cd0, t2.large, 500 GiB EBS `gp2` root storage).
```